src/core/qfieldcloudprojectsmodel.cpp

```diff
@@ -318,28 +354,36 @@ QString QFieldCloudProjectsModel::layerFileName( const QgsMapLayer *layer ) cons
   return layer->dataProvider()->dataSourceUri().split( '|' )[0];
 }
 
-QFieldCloudProjectsModel::PackagingStatus QFieldCloudProjectsModel::packagingStatus( const QString &status ) const
+QFieldCloudProjectsModel::JobStatus QFieldCloudProjectsModel::jobStatus( const QString &status ) const
```

Comment:
Maybe the function could have a better name, something like getStringJobStatus (to match your getStringJobType name below).

Secondary point, should you also move the getStringJobType to be a function of the class instead of a function-within-a-function? Merely asking.

Reply:
1) renamed to `getJobStatusFromString` and `getJobTypeAsString`.
2) addressed, too much scripting sugar here.

Reply:
@suricactus , if you want to stick to the established convention in QGIS, there, it is:
- encode[Whatever]( Enum ) to go from an enum to string
- decode[Whatever]( String ) to go from a string to an enum